tests/Aspire.Milvus.Client.Tests/AspireMilvusHelpers.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.XUnitExtensions;
+using Milvus.Client;
+
+namespace Aspire.Milvus.Client.Tests;
+public class AspireMilvusHelpers
+{
+    public const string TestingEndpoint = "http://localhost:19530";
```

Comment:
Let's not hard code this into the tests. Instead, get the `MilvusContainer` (test container) instance and ask it for the connection string.

Reply:
File no longer needed now that i added testcontainers...will delete.